server/src/data/addMedia.js

```diff
@@ -0,0 +1,104 @@
+const joi = require('@hapi/joi')
+
+const { ImageTypes } = require('../enums')
+
+const paramSchema = joi
+    .object()
+    .keys({
+        userId: joi
+            .string()
+            .uuid()
+            .required(),
+        imageType: joi
```

Comment:
We call it `appMedia`, so maybe we could make this more generic as well. For example, it would be possible to upload a video show case of the app as app media.

Reply:
True, but calling it appMediaType could be confusing as well because we also have a mediaType (mime) table.

Reply:
Just `type`? It's in the context of the `AppMedia` so being generic here should be fine.

Reply:
Or `mediaType`?

Reply:
Maybe its more appropriate to name this mediaType and then rename the existing mediaType to mimeType instead?

Reply:
Agreed, better to rename to `mediaType` and `mimeType`. Good point about `type` being reserved as a keyword as well.

Reply:
I don't think it is reserved in javascript, but I usually avoid it since it often is in other languages. Anyhow, I'll rename it as we agree on this!